frontend/src/components/EmailContact.js

```diff
@@ -0,0 +1,34 @@
+import styled from "styled-components";
+
+const ContactContainer = styled.div`
+  margin: -80px 20px 80px;
```

Comment:
A -80px felső margó helyett gondolom jobb lenne a felette lévő kártya lista alsó paddingjét 110px-ről 30px-re módosítani, de az jelenleg külön branch-en van:
[CardListStyled.js](https://github.com/flowacademyhu/piac-app/blob/technical-233/frontend/src/styles/CardListStyled.js)

Reply:
Igen, itt valójában sok probléma gyűrűzött be. Ha ilyennel találkozol, akkor célszerű a "five whys" módszert használni (https://en.wikipedia.org/wiki/Five_whys), hogy a probléma gyökerét megtaláld és ennek megfelelően old meg.

1. Miért -80px a marginja ennek a komponensnek? Mert ahol használjuk ott a fölötte lévő elemnek (card list) van egy plusz 110px-es alsó paddingja.
2. Miért van a card listnek 110px-es alsó paddingja? Hogy ne lógjon rá a footer.
3. Miért lóg rá a footer? Mert a footer position fixed-es és ha nincs a legutolsó elem, a card list kipaddolva alul, akkor rálóg.
4. Miért a card list van kipaddolva? Mert mindenhol a card list az utolsó elem, minden oldalon.
5. Miért pont a card list az utolsó elem? Ez a véletlen műve, most így van, de valójában más elem is lehetne az utolsó.

És akkor ebből el is jutunk a megoldáshoz: valójában nem kell a negatív margó, ahogy a card listen a plusz padding sem, hanem az oldalnak kellene olyan vázat adni, ahol a görgethető tartalom és a fixed footer nem zuhan össze. Praktikusan én azt csinálnám, hogy a body-ra annyi paddingot raknék alulra, hogy semmilyen tartalmat ne tudjon eltakarni a footer, így sem a contact containernek, se a card listnek nem kell plusz-mínusz padding-margin. Vagy ha lokalizálni akarjuk kicsit a dolgot, akkor ne is a body kapjon alsó paddingot, hanem az a rész, ahol összeáll az oldal layout-ja a gördíthető tartalommal és a footerrel az App.js-en belül. Ott lehetne azt csinálni, hogy egy content div-et körérakni a routernek és a footert utána szúrni, a content divet meg kipaddolni. Ez jobban egységbezárt módosítás, mint  ha globálisan a body kap paddingot.
**Szerintem ez menjen be így** amellett, hogy felveszel egy issuet erre a problémakörre és ha kész ez a story meg a styled-components-es, akkor ki lehet javítani.